include/arpa/inet.h

```diff
@@ -23,9 +23,9 @@
 int inet_aton(const char *cp, struct in_addr *inp);
 in_addr_t inet_addr(const char *cp);
 char *inet_ntoa(struct in_addr in);
-const char *inet_ntop(int af, const void *restrict src,
-		       char *restrict dst, socklen_t size);
-int inet_pton(int af, const char *restrict src, void *restrict dst);
+const char *inet_ntop(int af, const void *src,
```

Comment:
FYI, `restrict` keyword is in the Open Group POSIX specification for this function: https://pubs.opengroup.org/onlinepubs/9699919799/functions/inet_ntop.html

Were the keywords removed due to the implementation?